docs/contributing/how-to-release.md

```diff
@@ -2,14 +2,19 @@
 
 Creating a new release requires a few things before getting started:
 
-* Linux or OSX with 7z installed.
-* Windows with HTML Help Workshop installed.
-* Doxygen 1.8.8, and GraphViz installed on both machines.
+* Linux or OSX.
+* Windows with HTML Help Workshop, and Inno Setup installed.
+* 7-Zip, Doxygen 1.8.8, and GraphViz installed on both machines.
 
 Unless mentioned otherwise, all steps should be run on Linux or OSX so that the
 repository export used for the release is primarily using LF line endings. Files
 that require CRLF line endings will be converted appropriately.
 
+*Important:* Ensure that 7-Zip, HTML Help Workshop, Doxygen, GraphViz and Inno
+Setup have all been added to your Path in Windows. You can confirm this by
+running `7z`, `hhc`, `iscc`, `doxygen -v`, and `dot -V` in a command prompt.
+Add the missing installed folder locations of any executables to your Path.
```

Comment:
Definitely.

OTOH I think it would be nice to actually check that all the necessary tools are available from the batch file itself because while I definitely know how to set the PATH, I can (and will) forget to do it and it would be helpful to be told about this immediately. I think it can be done using `where.exe /q` but I didn't test it...